redis/commands.hh

```diff
@@ -39,6 +39,18 @@ public:
     virtual future<redis_message> execute(service::storage_proxy&, redis_options&, service_permit) override;
 };
 
+class exists : public abstract_command {
+    std::vector<bytes> _keys;
+    size_t _count = 0;
+public:
+    static shared_ptr<abstract_command> prepare(service::storage_proxy& proxy, request&& req);
+    exists(bytes&& name, std::vector<bytes>&& keys)
+        : abstract_command(std::move(name))
+        , _keys(std::move(keys)) {
+    }
```

Comment:
Let's move the constructor definition to the `commands.cc` file to keep the implementation in one place.